In [ ]:
import numpy as np
import cv2 as cv
import matplotlib.pyplot as plt

img = cv.imread('Images/ER.jpg')

px = img[100:105,100:105] # 5x5 block of pixel, starting from [100,100]
# remember, [B G R] in that order
px

array([[[ 45,  80, 183],
        [ 43,  78, 181],
        [ 43,  78, 181],
        [ 43,  78, 181],
        [ 48,  83, 186]],

       [[ 46,  81, 184],
        [ 44,  79, 182],
        [ 43,  78, 181],
        [ 43,  78, 181],
        [ 47,  82, 185]],

       [[ 45,  83, 185],
        [ 43,  81, 183],
        [ 43,  81, 183],
        [ 43,  81, 183],
        [ 44,  82, 184]],

       [[ 46,  84, 186],
        [ 45,  83, 185],
        [ 45,  83, 185],
        [ 45,  83, 185],
        [ 44,  82, 184]],

       [[ 47,  85, 187],
        [ 45,  83, 185],
        [ 40,  78, 180],
        [ 37,  75, 177],
        [ 45,  83, 185]]], dtype=uint8)

[ 45,  80, 183]: reddish, because R has the highest value

5 rows x 5 columns -> 25 pixels total
each pixel has 3 values, so the result is 5x5x3 array

array([
  [ [B, G, R], [B, G, R], ..., ],
  [ [B, G, R], [B, G, R], ..., ],
  ...
], dtype=uint8)
